packages/wonder-blocks-dropdown/components/dropdown.test.js

```diff
@@ -0,0 +1,98 @@
+//@flow
+import React from "react";
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
+
+import OptionItem from "./option-item.js";
+import Dropdown from "./dropdown.js";
+
+const keyCodes = {
+    tab: 9,
+    escape: 27,
+    space: 32,
+    up: 38,
+    down: 40,
+};
+
+describe("Dropdown", () => {
+    global.scrollTo = jest.fn();
+    global.getComputedStyle = jest.fn();
+
+    afterEach(() => {
+        unmountAll();
+    });
+
+    it("handles keyboard behavior as expected", () => {
+        const onOpen = jest.fn();
+
+        const dummyOpener = mount(<div />);
+
+        // A shell of a dropdown that's open
+        const dropdown = mount(
+            <Dropdown
+                initialFocusedIndex={0}
+                // mock the itemrefs
+                items={[
+                    {
+                        component: (
+                            <OptionItem label="item 1" value="1" key="1" />
+                        ),
+                        focusable: true,
+                        populatedProps: {},
```

Comment:
We should probably have a test that checks that the callbacks that can appear in `populatedProps` will get called at the right time.